pkg/daemon/rpm-ostree.go

```diff
@@ -395,55 +437,58 @@ func runGetOut(command string, args ...string) ([]byte, error) {
 	return rawOut, nil
 }
 
+type Deployment struct {
```

Comment:
It's less gross, and it's not an array anymore, but it's still here! :smile:

(This is just commentary, I assume we'll leave this here until we decide whether to break some of this out into some sort of library/bindings or not)

Reply:
Haha yep just needed to have Deployment vars